api/api.go

```diff
@@ -29,13 +30,6 @@ func (e Error) Error() string {
 	return e.Message
 }
 
-type ScannerStats struct {
-	BytesRead      int64 `json:"bytes_read"`
-	BytesMatched   int64 `json:"bytes_matched"`
-	RecordsRead    int64 `json:"records_read"`
-	RecordsMatched int64 `json:"records_matched"`
```

Comment:
This change alters the names of these fields in ZJSON responses because zbuf.ScannerStats doesn't have these struct field tags.